### DISCUSSION TITLE
Circulation pump

### DISCUSSION THREAD

**Discussion by alexdelli** [raomin/ESPAltherma#264] [Q&A] [answered]
Sorry, this is not an ESPAltherma issue. It is just an info looking at ESPAltherma data:

Why the circulation pump is something on during the night, if the water temp is set to  off during the night?
I set the water temp from 22:00 to 6:00 at 30°, and after at 46°.
But during the night if the water temp is 40°, the circulation pump every 10 minutes goes on (but without heating the water)
is it normal?
thank you

**Comment by fellek** [comment]
Hi, as far as I know, this should be expexted/normal behavior. The water has to circulate. if the water temp drops under your wanted temp of 30° it would heat again. I think this behvior is mentioned in the Manuals for my Altherma.
I hope this is useful...

**Comment by raomin** [comment]
Mine is always between 20 and 28l/min. I'm based on an external thermostat. So I believe it should not be needed to circulate 24/7.
Is there a setting where to change this?

**Reply by Tardymo** [reply, marked as answer]
So your external thermostat switches heating off, but the pump is still running 24/7? If so, then yes, there is a setting. On my heatpump it is under menu 4.5 (Space Heating/Cooling -> Pump operation mode). You have it on continuous, but this can be set to Request.

**Reply by raomin** [reply]
Works! Big electric savings! Thanks!
